workflow/rocoto/gefs_tasks.py

```diff
@@ -339,11 +340,17 @@ def wavepostsbs(self):
                      'maxtries': '&MAXTRIES;'
                      }
 
-        member_var_dict = {'member': ' '.join([str(mem).zfill(3) for mem in range(0, self.nmem + 1)])}
-        member_metatask_dict = {'task_name': 'gefs_wave_post_grid',
-                                'task_dict': task_dict,
-                                'var_dict': member_var_dict
-                                }
+        fhrs = self._get_forecast_hours('gefs', self._configs['wavepostsbs'])
```

Comment:
Okay, there are a couple issues here:
1. [_get_forecast_hours()](https://github.com/AntonMFernando-NOAA/global-workflow/blob/57c8aa36ed6ae9a32797b0af0a8029fd843b99e3/workflow/rocoto/tasks.py#L135) doesn't take in a dict, just a string denoting the component
2. `_get_forecast_hours()` hasn't been updated to produce the correct forecast hours for waves using `FHOUT_WAV`, `FHOUT_HF_WAV`, etc.

Reply:
NB: I don't think there is wave output at the start, so you will need to remove `0` from the list like ocean and ice does.

Reply:
@WalterKolczynski-NOAA @AntonMFernando-NOAA there is output in GFSv16 wave for forecast hour 0.

Reply:
I  now see this is GEFS, but I did just check ops for that and I also see f000 for GEFS.